### DISCUSSION TITLE
Codes COnfusions

### DISCUSSION THREAD

**Discussion by Devansh-Bots** [Devansh-Bots/Mdisk_downloader-main#5] [Q&A] [answered]
Why is java and python built different?

**Comment by noob-mukesh** [comment, marked as answer]
There is more experimentation than production code. Java is a statically typed and compiled language, and Python is a dynamically typed and interpreted language. This single difference makes Java faster at runtime and easier to debug, but Python is easier to use and easier to read.

**Reply by Devansh-Bots** [reply]
Okay, nice answer!

**Comment by Putzzrowr** [comment]
Lemzzxx
